public/app/plugins/datasource/prometheus/partials/annotations.editor.html

```diff
@@ -27,4 +27,11 @@ <h5 class="section-heading">Field formats</h6>
 			</div>
 		</div>
 	</div>
+
+	<h5 class="section-heading">Other options</h5>
+	<div class="gf-form-inline">
+		<div class="gf-form">
+			<gf-form-switch class="gf-form" label="Series value as timestamp" label-class="width-10" checked="ctrl.annotation.seriesValueAsTimestamp">
```

Comment:
width should be 14 to avoid wrapping the label

Reply:
We should also introduce some kind of tooltip for explaining this feature a bit better and make sure to mention that the user needs to return the value in ms and therefore most likely multiply it with 1000 since seconds is the convention for times in Prometheus.

Reply:
I fix and improve these point.